plugins/modules/packaging/os/yum_versionlock.py

```diff
@@ -125,18 +140,11 @@ def main():
     packages_list = []
     if state in ('present'):
```

Comment:
That this actually worked is more an accident:
```suggestion
    if state in ('present', ):
```